atom/browser/native_window.cc

```diff
@@ -317,6 +317,17 @@ void NativeWindow::InspectElement(int x, int y) {
   agent->InspectElement(x, y);
 }
 
+void NativeWindow::InspectServiceWorker() {
```

Comment:
Why is `BrowserWindow.inspectServiceWorker` needed? I think we can just use `window.webContents.inspectServiceWorker`.